src/main/groovy/com/avioconsulting/mule/linter/model/Rule.groovy

```diff
@@ -0,0 +1,60 @@
+package com.avioconsulting.mule.linter.model;
+
+class Rule {
+    String ruleId
+    String ruleName
+    RuleSeverity severity = RuleSeverity.MINOR
+    Application application
```

Comment:
Should a Rule itself have a reference to the application?  The application isn't part of the definition of a rule itself.

Reply:
Yes... the rule knows which files in an application to run on... 

Is there a better way to do this?